agent/src/main/java/com/appland/appmap/transform/ClassFileTransformer.java

```diff
@@ -102,9 +102,12 @@ private List<Hook> getHooks(String methodId) {
   }
 
   private void processClass(CtClass ctClass) {
+    logger.trace(() -> ctClass.getName());
     for (CtBehavior behavior : ctClass.getDeclaredBehaviors()) {
+      logger.trace(() -> behavior.getLongName());
       Hook hook = Hook.from(behavior);
       if (hook == null) {
+        logger.trace("{}, no hooks", () -> behavior.getLongName());
```

Comment:
I take it the logging additions have been left in purposefully? :+1:

Reply:
Yeah, they were helpful in figuring why `SpringServletContainerInitializer` wasn't getting hooked.

One of the nice features of tinylog is this ability to pass a lambda. It only gets evaluated if the message is actually going to get logged, so these calls are really cheap.